vpu/generic/vpuGeneric/.conf/shareWSLPorts.ps1

```diff
@@ -0,0 +1,58 @@
+# DOES NOT ADD THE suppress HERE
+# THIS NEED TO BE SUPPORTED ON THE WINDOWS POWERSHELL
```

Comment:
`# THIS NEEDS TO BE SUPPORTED ON THE WINDOWS POWERSHELL`

Typo, again, probably a shared script.

Reply:
I will fix it in this PR and we can fix this also at the source later.

Reply:
Actually all this typos are on the scripts automatically copied from the [vscode-torizon-templates scripts](https://github.com/toradex/vscode-torizon-templates/tree/dev/scripts). Abdur, can you submit the typo fixes on the [vscode-torizon-templates](https://github.com/toradex/vscode-torizon-templates) repo (on the dev branch), please?

Reply:
@andreriesco sure, I will do it.

Reply:
```
% find . -name "*shareWSLPorts*"
./torizon-samples/tflite/python/tflitePythonVivante/.conf/shareWSLPorts.ps1
./torizon-samples/pwm/C/pwmC/.conf/shareWSLPorts.ps1
./torizon-samples/i2c/python/mpu6050Python/.conf/shareWSLPorts.ps1
./torizon-samples/spi/C/spiC/.conf/shareWSLPorts.ps1
./torizon-samples/bluetooth/c/.conf/shareWSLPorts.ps1
./torizon-samples/gpio/python/gpioPython/.conf/shareWSLPorts.ps1
./torizon-samples/gpio/dotnet/gpioDotnet/.conf/shareWSLPorts.ps1
./torizon-samples/gpio/C/gpioC/.conf/shareWSLPorts.ps1
./scripts/shareWSLPorts.ps1
```
 Guys, in `torizon-templates` we seem to have a good mechanism to re-use these scripts. The duplication is happening here.

Reply:
On the `vscode-torizon-templates` the scripts are added to the project when it is created, but this is done by the Torizon Extension on the creation of the project. Here the samples are used a bit differently, as a ready-to-use workspace folder that the user clones and uses.
It would be nice to be able to list and create a sample in a similar way that it is done with a project inside VSCode, but some investigation on how to nicely integrate the samples there is needed (I will check if there is a ticket on Jira to do it).

Reply:
@andreriesco there is TIE-853

Reply:
I'll resolve this as it's tracked somewhere else, thanks.